CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Unreleased changes, in reverse chronological order. New entries are added at the top of this list.
 
+- [#1511](https://github.com/Zilliqa/zq2/pull/1511): Add more API tests.
```

Comment:
```suggestion
```
I've usually been leaving non-user-visible changes out of the changelog. But happy to do otherwise if others think we should track them.